packages/contract-types/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "es2019",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "outDir": "dist",
+    "resolveJsonModule": true
+  },
+  "lib": ["es2019"]
+}
```

Comment:
you can use the node 20 preset:

```
{
  "extends": "@tsconfig/node20/tsconfig.json",
  "compilerOptions": {
    "strict": true,
    "resolveJsonModule": true,
    "sourceMap": true,
    "outDir": "dist"
  }
}
```

either way, most important is to add `strict: true`

Reply:
^ i'll add it in a follup. merging now

Reply:
fixed in https://github.com/daimo-eth/daimo/pull/24